module/FileNVRAM.c

```diff
@@ -359,7 +388,18 @@ static void readplist()
                 
                 if (plistSize && read(fh, plistBase, plistSize) == plistSize)
                 {
-                    XMLParseFile( plistBase, &gPListData );
+                    // cleaning nvram.plist from \n and \t.
+                    // data tag have those bytes after the serialization:
+                    //
+                    //  <key>SystemAudioVolume</key>
+                    //  <data>
+                    //      fw==
+                    //  </data>
+                    //
+                    // but I want it to be: <data>fw==</data>
+                    //
+                    // so some garbage ends up in /chosen/nvram. just remove them!
+                    XMLParseFile(strdelchar(strdelchar(plistBase, '\n'), '\t'), &gPListData);
```

Comment:
Is this here due to a bug in chameleon?
If so, I'd suggest noting that and preferably fixing chameleon. I don't mind having a workaround here, but please make sure chameleon is fixed as well.